apps/web/components/booking/pages/AvailabilityPage.tsx

```diff
@@ -303,7 +303,7 @@ const AvailabilityPage = ({ profile, eventType, ...restProps }: Props) => {
     () => <TimezoneDropdown timeZone={timeZone} onChangeTimeZone={setTimeZone} />,
     [timeZone]
   );
-  const stripeAppData = getStripeAppData(eventType);
+  const paymentAppData = getPaymentAppData(eventType);
```

Comment:
This should fetch eventType current enabled payment app.
@TODO: Should only allow one payment app per event type.